packages/ethereum-light-client/src/config.rs

```diff
@@ -0,0 +1,382 @@
+use core::{
```

Comment:
This file is copied in from union for convenience, but it should be cleaned up or removed eventually. Tracking that in https://github.com/cosmos/solidity-ibc-eureka/issues/147